samples/stock/demo/two-standalone-navigators/demo.js

```diff
@@ -0,0 +1,124 @@
+const commonOptions = {
+    api: {
+        url: 'https://demo-live-data.highcharts.com',
+        access: {
+            url: 'https://demo-live-data.highcharts.com/token/oauth',
+            token: 'token'
+        }
+    }
+};
+
+const AMDISIN = 'US0079031078';
+
+// eslint-disable-next-line no-undef
```

Comment:
Avoid lint exceptions in the demo code. Instead, try adding a `global` statement in `.eslintrc`, either at the root of this specific demo, or in the samples directory.

Reply:
+1, @bm64 have a look how it's done here: https://github.com/highcharts/connectors-morningstar/blob/b512f475b95236452dc8622097d532c7f4e55cd8/.eslintrc#L12